pytorch_lightning/callbacks/pt_callbacks.py

```diff
@@ -355,11 +363,11 @@ def on_epoch_begin(self, epoch, trainer):
                 break
 
 
-if __name__ == '__main__':
-    c = EarlyStopping(min_delta=0.9, patience=2, verbose=True)
-    losses = [10, 9, 8, 8, 6, 4.3, 5, 4.4, 2.8, 2.5]
-    for i, loss in enumerate(losses):
-        should_stop = c.on_epoch_end(i, logs={'val_loss': loss})
-        logging.info(loss)
-        if should_stop:
-            break
+# if __name__ == '__main__':
```

Comment:
why comment out?

Reply:
I do not think that there should be any `__main__` in the package